src/components/Icons/styled.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import { Link } from "react-router-dom";
+
+export const IconLink = styled(Link)`
+  color: var(--red);
+  font-size: calc(1em + 8vmin);
+  filter: drop-shadow(1px 1px 1px #000);
+  fill-rule: #fff;
+  cursor: pointer;
+  transition: all 0.2s;
+  &:hover {
+    transform: scale(1.2);
+  }
+`;
+
+export const RulesLink = styled.div`
+  color: var(--red);
+  font-size: calc(1em + 8vmin);
+  filter: drop-shadow(1px 1px 1px #000);
+  fill-rule: #fff;
+  cursor: pointer;
+  transition: all 0.2s;
```

Comment:
The scrollbar pops up for me when the hover transformation occurs. You can fix this by making the `transform-origin` the bottom right, or setting `overflow: hidden` on the App component.